docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - kibana
 
   api:
-    build: ./lab/api
```

Comment:
@ubijco 
this hs to be changed in the Dockerfile: ./lab/api/Dockerfile

Reply:
Oh my bad, did not realize that was a link.  Let me fix that and re-submit.